### ISSUE TITLE
Reviewing the structure of the ImSwitch installer

### ISSUE THREAD

**Issue by beniroquai** [beniroquai/ImSwitchInstaller#1]
Hey @jacopoabramo, as I said, I was working on a conda constructor-based installer for ImSwitch. It seems it works on Windows after some trial and error. General idea: Install miniconda locally, then pull ImSwitch (right now via github api, download latest push - could be release or anything), and install it with the conda environment created with the constructor. It also creates a symlink on the desktop and installs additional drivers (e.g. for UC2 stuff). The main stuff happens in `install.py`. What do you think? Makes sense? With this, the release is ~60Mb instead of >1GB and mostly represents a one-click installer rather than a copy paste of many commands in the commandline.

**Comment by jacopoabramo** [comment]
Can this be automated via a release so that people can download the final installer? Via a GitHub action or something. I imagine that people would just want to have the final one-click installer rather than going through all that process

**Comment by beniroquai** [comment]
Certainly. Once the installer works, this will be the result. Under release you can find an `.exe` that also creates a double-click start imswitch thingy. But conda behaves really weird...Would need a fresh system to test if it actually starts - could even be an action running in CLI?

**Comment by jacopoabramo** [comment]
Very cool, we can probably change the current GitHub actions so that whenever we trigger a new release tag we also build this up
